.evergreen/functions.yml

```diff
@@ -202,6 +202,11 @@ functions:
 
           # Install dependencies
           bash ".evergreen/retry-with-backoff.sh" .evergreen/npm_ci.sh
+          # Will fail if versions of direct dependencies listed in package-lock
+          # are not matching versions defined in package.json file of any of the
+          # workspace packages
+          # TODO: change this to npm ls --all when rest of the mismatched version issues are resolved
```

Comment:
No ticket because I'll just do it right away (or will add a ticket if resolving the issue for transitive deps will turn out to take longer than expected)